tests/test_other.py

```diff
@@ -8325,6 +8325,11 @@ def test_wasm_target_and_STANDALONE_WASM(self):
         # verify the wasm runs with the JS
```

Comment:
Would it make sense to move this test to test_core.py.. given that its running the test in all the different opt levels?

Reply:
It's an awkward fit for `core` atm since core handles building for you, including picking what file to emit (either `src.cpp.o.js` or `src.c.o.js`). Changing that might be a large refactoring.

Reply:
Does it?  Can you call `self build()` or even `run_process(EMCC...`  just fine from core?   The only think you need to remember to do is include  `self.get_emcc_args()` I think.

Reply:
Yeah, you can call the internals instead of using the main `do_run*` functions, I guess. It's less elegant though as the code in `other` for such things is cleaner, where there are no assumptions about a "standard" model.

But yeah, maybe ignoring all the standard model stuff and just calling `self.get_emcc_args()` would work ok. I don't think we do that in any tests yet though, except maybe one or two.